RUFAS/input_manager.py

```diff
@@ -726,6 +727,33 @@ def get_data_keys_by_properties(self, target_properties: str) -> list[str]:
 
         return data_keys
 
+    def delete_data(self, data_address: str) -> bool:
```

Comment:
A couple things here:
- Please add docstrings.
- I think it would be helpful as well to expand the function name to encompass that it deletes both the input data and the corresponding metadata - `delete_input_and_metadata` for example.